Package.swift

```diff
@@ -48,7 +48,7 @@ let package = Package(
     targets: [
         .target(
             name: "WalletConnectSign",
-            dependencies: ["WalletConnectPairing"],
+            dependencies: ["WalletConnectPairing", "WalletConnectVerify"],
```

Comment:
As discussed I would not make WalletConnectSign dependent from WalletConnectVerify
but WalletConnectNetworking instead